File: GazeChim.Data/Models/BsffDestination.cs
using GazeChim.Data.Models.Enums;

namespace GazeChim.Data.Models
{
    public class BsffDestination : BsffCompany
    {
        public BsffOperationCode PlannedOperationCode { get; set; }
        public BsffReception Reception { get; set; }
        public string Cap { get; set; }
    }
}

File: GazeChim/ClientApp/src/app/models/bsff/bsff-destination.ts
import { BsffCompany } from './bsff-company';
import { BsffReception } from './bsff-reception';
import { BsffOperationCode } from './enums/bsff-operation-code';

export interface BsffDestination extends BsffCompany {
  plannedOperationCode: BsffOperationCode;
  reception: BsffReception;
  cap: string;
}
